### DISCUSSION TITLE
Synchronizing the Browser

### DISCUSSION THREAD

**Discussion by angtianqiang** [David-Moreira/BlazorXTabs#46] [Q&A] [answered]
Can it be combined with navigation, such as when the current tab is active, the browser address can also be changed synchronously

**Comment by David-Moreira** [comment]
Hello.
It does open the tab on navigation similar to how a browser would work. 
Try clicking the side tab to navigate. 
https://david-moreira.github.io/BlazorXTabs/

Is this what you mean?

**Comment by angtianqiang** [comment]
![image](https://user-images.githubusercontent.com/10491196/206939413-2b6ecd68-c32a-46a7-ace0-fc42cb98ff33.png)

When I activate the counter page with my mouse, the browser should display this:
https://david-moreira.github.io/BlazorXTabs/counter

**Reply by David-Moreira** [reply]
Ah I see what you mean. No currently that's not supported. To synchronise the browser address it would have to retrigger a navigation. It could possibly be an opt in feature I'll look into it as soon as I have time.

**Reply by angtianqiang** [reply]
Thank you very much, you can refer to this project under https://github.com/liningit/BlazorWebassemblyMultiPagesTabNoIframe

**Reply by David-Moreira** [reply, marked as answer]
Hello @angtianqiang 
Sorry that it took a while, the feature has been published.
You may use the new **NavigationMode.Navigable** to set it so that when a Page Tab is set to active, navigation happens, and the browser url is synced.

You can now find it working on the demo.

**Reply by angtianqiang** [reply]
TKS ,
